### DISCUSSION TITLE
moving pivpn

### DISCUSSION THREAD

**Discussion by hs6666** [pivpn/pivpn#1842] [Support]
hi, my pi running pivpn is getting very old :( if i was to fire up a debian instance is there a way of just installing and copying over the config? (then then switch off the pi and give the debian vm my pi's old ip address)

**Comment by orazioedoardo** [comment]
If it's very old you are better off reinstalling from scratch

**Comment by hs6666** [comment]
i've actualy discovered the "joys" of tailscale.... so easy to setup, especially if like me you dont need "specific user based login vpn"
